2nd task to find grade of student/script.js

```diff
@@ -0,0 +1,83 @@
+//PROMPT METHOD
+
+let marks = prompt("ENTER YOUR MARKS :")
+
+if(marks >= 80){
+    document.write("distinction")
+}
+else if(marks >= 60 && marks < 80) {
+    document.write("first division");
+} 
+else if (marks >= 50 && marks< 60) {
+    document.write("second division");
+} 
+else if (marks >= 40 && marks <50) {
+    document.write("third division");
+}
+else if(marks<40) {
+    document.write("you're failed");
+}
+
+
+
+
+
+
+//ARRAY METHOD
+
+let scores = [30, 66, 96, 56, 67, 45, 88, 76, 24, 69]
```

Comment:
lets use const for this array declaration since the data in array are not going to change